.travis.yml

```
@@ -1,17 +1,21 @@
+os: linux
+dist: bionic
 language: python
 python:
   - 2.7
   - 3.5
   - 3.6
   - 3.7
   - 3.8
+services:
+  - docker
 
 before_install:
 - pip install --upgrade pip
 - pip install flake8  # find Python syntax errors and undefined names
 - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
 install:
-- docker pull ibmcom/db2express-c
+- docker pull ibmcom/db2
 - docker run --name db2  -p 50000:50000 -e DB2INST1_PASSWORD=password -e LICENSE=accept -d ibmcom/db2express-c db2start
```

```suggestion
- docker run --name db2 -itd --privileged=true -p 50000:50000 -e LICENSE=accept -e DB2INST1_PASSWORD=password -e DBNAME=sample -v $HOME/database:/database ibmcom/db2
```

Looking at the Quickstart section of the [docs](https://hub.docker.com/r/ibmcom/db2) I think this needs to change as well